addon/components/frost-about-dialog.js

```diff
@@ -1,19 +1,63 @@
+/**
+ * A component for rendering a frosty about dialog
+ */
+
 import Ember from 'ember'
+import PropTypeMixin, {PropTypes} from 'ember-prop-types'
+
 import layout from '../templates/components/frost-about-dialog'
 
 const {Component, inject} = Ember
 
-export default Component.extend({
+export default Component.extend(PropTypeMixin, {
```

Comment:
Yeah, found that out after it was merged. I had checked Bunsen, which was still using the mixin too